src/modules/powerrename/ui/PowerRenameUI.cpp

```diff
@@ -429,6 +469,13 @@ void CPowerRenameUI::_OnInitDlg()
         RegisterDragDrop(m_hwnd, this);
     }
 
+    RECT rc = { 0 };
+    GetWindowRect(m_hwnd, &rc);
+    m_initialWidth = RECT_WIDTH(rc);
```

Comment:
It would be nice to save the size and restore it. But this can be for a future update.
It would also be nice to resize the ListView columns when resizing the dialog.

Reply:
Thanks!  I will add that to my backlog. :)